c10/util/SmallVector.h

```diff
@@ -376,9 +373,9 @@ class SmallVectorTemplateCommon
 /// note
 template <
     typename T,
-    bool = (std::is_trivially_copy_constructible<T>::value) &&
-        (std::is_trivially_move_constructible<T>::value) &&
-        std::is_trivially_destructible<T>::value>
+    bool = (std::is_trivially_copy_constructible_v<T>)&&(
+        std::is_trivially_move_constructible_v<
+            T>)&&std::is_trivially_destructible_v<T>>
```

Comment:
Was clang-format not run on this file? Seems like odd style not have spaces around &&

Reply:
The current lower version of clang-format encourages this style...

Reply:
https://github.com/pytorch/pytorch/pull/116002 <- hope this will help